lib/appsignal.ex

```diff
@@ -115,10 +115,11 @@ defmodule Appsignal do
   When there is no current transaction, this call starts one.
 
   """
-  def send_error(reason, message \\ "", stack \\ nil, metadata \\ %{}, conn \\ nil) do
+  def send_error(reason, message \\ "", stack \\ nil, metadata \\ %{}, conn \\ nil, fun \\ fn(t) -> t end) do
```

Comment:
It would be nice to improve the documentation of this function. But you can also open a separate issue for this.